NUModels/NUCategory.j

```diff
@@ -51,24 +51,24 @@
 
 - (id)initWithName:(CPString)aName
 {
+    [self initWithName:aName contextIdentifier:nil filter:nil];
```

Comment:
```return [self initWithName:aName contextIdentifier:nil filter:nil];```
should be enough.

Also, if we go for a mandatory context, we will need to update this constructor